diff --git a/node_modules/react-native-date-picker/android/build.gradle b/node_modules/react-native-date-picker/android/build.gradle
index abe1003..f19968b 100644
--- a/node_modules/react-native-date-picker/android/build.gradle
+++ b/node_modules/react-native-date-picker/android/build.gradle
@@ -26,6 +26,6 @@ dependencies {
     implementation fileTree(include: ['*.jar'], dir: 'libs')
     implementation 'com.facebook.react:react-native:+'
     implementation 'com.henninghall.android:NumberPickerView:1.1.1'
-    implementation 'org.apache.commons:commons-lang3:3.6'
+    implementation 'org.apache.commons:commons-lang3:3.7'
     implementation group: 'net.time4j', name: 'time4j-android', version: '4.2-2018i'
 }
